common/src/main/java/com/wynntils/gui/widgets/BasicTexturedButton.java

```diff
@@ -60,4 +61,12 @@ public void onPress() {}
 
     @Override
     public void updateNarration(NarrationElementOutput narrationElementOutput) {}
+
+    public void editTooltip(List<Component> newTooltip) {
+        tooltip = ComponentUtils.wrapTooltips(newTooltip, 250);
+    }
+
+    public List<Component> getTooltip() {
+        return Collections.unmodifiableList(tooltip);
```

Comment:
What if we made the tooltip unmodifiable in `editTooltip`?

Reply:
This was actually just for debug and didn't need to exist